imports/plugins/core/accounts/client/helpers/templates.js

```diff
@@ -26,35 +26,3 @@ Template.registerHelper("displayName", function (displayUser) {
     }
   }
 });
-
-/*
- * registerHelper fName
- */
-Template.registerHelper("fName", function (displayUser) {
```

Comment:
Note that (a) the exact same helper was registered in two places, and (b) I chose to remove it rather than fix the lint error in it. I do not see where this helper is used anywhere.